src/UI/Switch.elm

```diff
@@ -0,0 +1,197 @@
+module UI.Switch exposing
+    ( Switch, switch
```

Comment:
What do you think about instead of having the color as a modifier, we use as an entry point?

so instead of `Switch.switch`, we can have:

- `Switch.default`
- `Switch.success`
- `Switch.danger`